drafts/te-topo-profile/draft-ietf-teas-te-topology-profiles.txt

```diff
@@ -659,21 +715,29 @@ Internet-Draft            TE Topology Profiles                April 2024
    1 or Option 3).  These attributes can be TE or non-TE and require the
    implementation of the te container.
 
-4.  Implemented profiles
+4.  Open Issues
 
-   When a server implements a profile of the TE topology model, it is
-   not clear how the server can report to the client the subset of the
-   model being implemented.
+4.1.  Supporting node/link versus overlay/underlay
 
+   Some more explanation of the difference between supporting-node/
+   supporting-link and overlay/underlay has been requested.
 
 
 
 
-Busi, et al.             Expires 21 October 2024               [Page 12]
+Busi, et al.             Expires 21 October 2024               [Page 13]
 
 Internet-Draft            TE Topology Profiles                April 2024
 
 
+   Note: that this issue is also tracked in github as issue #167.
+
+4.2.  Implemented profiles
+
+   When a server implements a profile of the TE topology model, it is
+   not clear how the server can report to the client the subset of the
+   model being implemented.
+
```

Comment:
Is it related to this "How to avoid the client to write an attribute not used in the TE Topology profile implemented by the serve" ? If yes better to report that, since this sentence on "how to report ..." does not give the view of the issue while the writing an attribute from client not implemented by the server is clear.

Reply:
The issue is generic but, while a work-around can be found for the read operations, the issue would be more severe for the write operation: text re-arranged to make it more clear